src/pages/contact/bubbles/ContactBubble.jsx

```diff
@@ -0,0 +1,14 @@
+const ContactBubble = (props) => (
```

Comment:
The Bubble is a not fitting the whole screen for me (see screenshot). This is a little however as a svg has defined width, heights and other property like the aspectRatio and quite some other weird ones affecting the size. Could you try to have to svg automatically fit the screen width? if this doesn't work we'll remove the bubble. Don't take to much time for this.
![grafik](https://user-images.githubusercontent.com/81356927/224482868-a8a0b4b1-a5e9-4e7d-922d-e9fd705aeaac.png)